third-party/rspamd/src/test/java/org/apache/james/rspamd/DockerRSpamD.java

```diff
@@ -0,0 +1,68 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.rspamd;
+
+import org.apache.james.rate.limiter.DockerRedis;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.utility.DockerImageName;
+
+public class DockerRSpamD {
+    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("a16bitsysop/rspamd");
+    private static final String DEFAULT_TAG = "3.2-r2-alpine3.16.0-r0";
+    private static final int DEFAULT_PORT = 11334;
+
+    private final DockerRedis dockerRedis;
+    private final GenericContainer<?> container;
+    private Network network;
+
+    public DockerRSpamD() {
+        this.network = Network.newNetwork();
+        this.dockerRedis = new DockerRedis(network);
```

Comment:
Redis?

Reply:
Doesn't start without it... Actually on all the images I tried (remember, there is no ifficial rspamd image) that's one I found that only required redis... others required other stuff

It seems that many modules of rspamd use redis, and that's probably why custom rspamd images all xpect a redis to start (from what I could test): 

We will use Redis as well anyway in our deployments, and we have already a docker redis image in james tests, so I thought was ok... If no then please find me one then, cause I spent a day last week swooping custom images

Reply:
That s ok